sysutils/flash/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+
+github.setup    hypriot flash 2.3.0
+
+platforms       darwin
+categories      sysutils
+license         MIT
+maintainers     @jrjsmrtn openmaintainer
```

Comment:
Since this is a script, add the following line here:

`supported_arches noarch
`

Reply:
Fixed.